webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric-association.js

```diff
@@ -89,7 +107,7 @@ class SakaiRubricAssociation extends RubricsElement {
           <div class="rubrics-selections">
             <select @change="${this.rubricSelected}" name="rbcs-rubricslist" aria-label="${tr("rubric_selector_label")}" class="form-control" ?disabled=${!this.isAssociated || this.readOnly}>
             ${this.rubrics.map(r => html`
-              <option value="${r.id}" ?selected=${r.id === this.selectedRubric}>${r.title}</option>
+              <option value="${r.id}" ?selected=${r.id == this.selectedRubric}>${r.title}</option>
```

Comment:
Is this change intentional? I always try to avoid coercion by using `===` rather than `==`

Reply:
It was intentional as that selectedRubric might have been coming from an attribute. I think I changed that though. Good spot.